src/attrs.rs

```diff
@@ -125,6 +125,34 @@ impl From<CacheMetrics> for Metrics {
     }
 }
 
+/// Font features for controlling typographic behavior
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub struct FontFeatures {
```

Comment:
This looks very good! I think it would be nice to document the font feature name for each of these, like `liga`